docs/content/config/security/understanding-the-ports.md

```diff
@@ -6,18 +6,29 @@ title: 'Security | Understanding the Ports'
 
 Prefer Implicit TLS ports, they're more secure and if you use a Reverse Proxy, should be less hassle (although it's probably wiser to expose these ports directly to `docker-mailserver`).
 
+!!! info "The Infamous Port 465"
+
+    Because there seems to be a lot of misconception and confusion when it comes to port 465, we would like to clarify some of the concerns that we frequently encounter in issues and discussions once and for all.
+
+    It is true that port 465 was historically assigned another protocol but then revoked. Nowadays (5 April 2023), it serves the ESMPT protocol for submission using Implicit TLS. So why do we recommend using port 465 (over 587)? It is - from a technical & security perspective - just better than port 587:
+
+    - Traffic on port 587 starts unencrypted but by issuing the STARTTLS command, new traffic will encrypted (afterwards); this is sub-optimal, because in a wrong configuration, TLS is not enforced! With DMS though, port 587 is correctly configured by default.
+    - Port 465 offers implicit TLS, i.e. traffic will always be encrypted, no matter what. This is superior to optional encryption.
+
+    **Please remember**: you will probably need to expose all of these ports (25 for transfer, 465 & 587 for submission) in order for your mail server to properly work with other mail servers!
```

Comment:
I don't think this should be covered right at the top, but we can better cover the issue to link users to.

I'll edit this page manually.